README.md

```diff
@@ -101,53 +122,111 @@ $ make
 $ make image
 ``` 
 
- 4. Create SRIOV network device plugin Pod
- ```
-$ kubectl create -f pod-sriovdp.yaml
+> See following sections on how to configure and run SRIOV device plugin.
+
+## Configurations
+
+### Config parameters
+This plugin creates device plugin endpoints based on the configurations given in file `/etc/pcidp/config.json`. This configuration file is in json format as shown below:
+
+```json
+{
+    "resourceList":
+    [
+        {
+            "resourceName": "sriov_net_A",
```

Comment:
@booxter 
> Idea for the future is making it possible to use the same root device for multiple pools, each pool having a separate device type, so that a user can request a VFIO enabled VF for one pod and a netlink device for another. (The VFIO pod can then unbind the device and register it with VFIO driver as needed.)

Interesting idea. We may need to revisit the VF discovery mechanism for each device types. With current implementation it is assumed, all VFs belong to a PF are registered with same driver -> same device type. In order to support multiple device types, we need to detect correct device type during VF discovery based on the driver attached to it.

> [I know the general recommendation was to pre-register devices with VFIO when cluster is deployed but it actually works fine without it if the pod then takes care of driver registration itself, like libvirt container in kubevirt does.)

To do this in Pod(or even with help of sriov-cni) it needs to be run as a privileged Pod. By having the device plugin to export correct vfio device information allows vfio device to be used in non-privileged Pod. We purposely stayed out of driver registration in the device plugins to avoid unnecessary complexities and many other issues around it.